makeservices/makemysql-real

```diff
@@ -155,85 +112,88 @@ def main():
             raise RuntimeError('Unable to read SUDO_USER.')
 
         # Read config file.
-        read_config()
+        mysql_host, mysql_root_pw = read_config()
 
         # Check whether the script should proceed.
-        intro_prompt()
+        if not intro_prompt():
+            print('>>> Aborted by user request.\n')
+            return
 
         # Connect to the MySQL server.
         try:
             print('>>> Connecting to MySQL database server...')
-            connection = MySQLdb.connect(host=MYSQL_HOST,
+            connection = MySQLdb.connect(host=mysql_host,
                                          user='root',
-                                         passwd=MYSQL_ROOT_PW)
+                                         passwd=mysql_root_pw)
         except:
-            print('>>> Error: Failed to connect to MySQL server.')
-            sys.exit(1)
+            print('>>> Error: Failed to connect to MySQL server.\n')
+            exit(1)
 
         # Check if the database already exists.
         try:
-            print(">>> Checking if database '%s' already exists..." % username)
+            print(">>> Checking if database '{}' already exists...".format(username))
             connection.select_db(username)
-            print('yes.')
+
+            # The database already exists, so skip the creation step.
+            print('yes.\n')
             db_create = False
             database_already_exists(username)
         except:
-            # The database already exists, so skip the creation step.
-            print('no.')
+            print('no.\n')
             db_create = True
 
         # Add or update user database privileges.
         try:
             userpass = os.popen(RANDOM_PW_CMD).read().strip()
```

Comment:
can we just generate the password in python rather than subprocess? the simpler this script the better

```python
import string, random
random_pw =''.join(random.SystemRandom().choice(string.ascii_letters + string.digits) for _ in range(20))
```

otherwise we should at least switch to subprocess